.eslintrc.js

```diff
@@ -54,6 +54,7 @@ module.exports = {
                 ],
                 "@typescript-eslint/explicit-function-return-type": "off",
                 "@typescript-eslint/no-non-null-assertion": "off",
+                "@typescript-eslint/no-explicit-any": "off",
```

Comment:
This rule is very inconvenient, developers can try to avoid using “any” as much as possible on their own.